index.html

```diff
@@ -8,47 +8,355 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
+    <title>Desktop version</title>
     <link rel="stylesheet" href="style.css" />
-    <title>Setup and mobile version</title>
   </head>
   <body>
-    <header>
-      <nav class="nav">
-        <h2 class="name">Mucyo</h2>
-
-        <i class="fa fa-bars"></i>
+    <div class="container">
+      <nav>
+        <div>Mucyo</div>
+        <div class="menu">
+          <ul>
+            <li><a href="#home">Home</a></li>
+            <li><a href="#news">News</a></li>
+            <li><a href="#contact">Contact</a></li>
+            <li>
+              <a href="#about">
+                <i class="fas fa-envelope"></i>
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="humberger">
+          <i class="fa fa-bars"></i>
+        </div>
       </nav>
-    </header>
-    <section class="headline-page">
-      <div class="introduction">
-        <h1 class="intro">Hey There.<br />I'm Mucyo</h1>
-        <h2 class="headline-page-h2">I'am a Software Developer</h2>
-
-        <p class="text-1">
-          I can help you build a product, feature or website <br />Look through
-          some of my work and experience!<br />
-          if you like what you see and have a project <br />
-          you need coded, don't hestiate to contacta me.
-        </p>
+      <main>
+        <div class="header">
+          <h1>Hey There.<br />I'm Mucyo</h1>
+          <h2>I'am a Software Developer</h2>
+
+          <p>
+            I can help you build a product, feature or website <br />Look
+            through some of my work and experience!<br />
+            if you like what you see and have a project <br />
+            you need coded, don't hestiate to contacta me.
+          </p>
+        </div>
+        <div id="social-icons">
+          <ul>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a>
+            </li>
+          </ul>
+        </div>
+      </main>
+      <div id="content_title">
+        <div class="my_recent">
+          <h1>My Recent Works</h1>
+        </div>
+        <div class="line1"></div>
+      </div>
+      <div id="sidebar">
+        <div class="txt_image">
+          <div class="img1">
+            <img src="images/Img.png" alt="image" class="middle-image" />
+          </div>
+          <div class="post1">
```

Comment:
> For this section, align both the left and right sides to the top.
![Screenshot from 2021-08-20 12-28-55](https://user-images.githubusercontent.com/65501535/130212842-efdb0696-254d-4195-8c83-10f86708a6fd.png)

Please consider aligning this section as previously requested.